solr/docker/templates/Dockerfile.official.header.template

```diff
@@ -90,5 +90,6 @@ RUN set -ex; \
   fi; \
   { command -v gpgconf; gpgconf --kill all || :; }; \
   rm -r "$GNUPGHOME"; \
-  tar -C /opt --extract --file "/opt/solr-$SOLR_VERSION.tgz";
+  tar -C /opt --extract --preserve-permissions --file "/opt/solr-$SOLR_VERSION.tgz"; \
+  find "/opt/solr-$SOLR_VERSION/" -type f -print0 | xargs -0 chmod 0644;
 
```

Comment:
Speaking of saving space, could we uninstall gpg at the end of this `RUN` command? I.e. `apt-get --purge remove gpg && rm -rf /var/lib/apt/lists/*;` at the end?

Reply:
Just tried this out and it didn't affect this size of the layer, so probably no need to do it.